meta-balena-common/recipes-containers/hostapp-extensions-update/hostapp-extensions-update.bb

```diff
@@ -0,0 +1,35 @@
+DESCRIPTION = "BalenaOS hostapp extensions updater"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+SRC_URI = " \
+	file://update-hostapp-extensions \
```

Comment:
Nitpick: the recipe is called `hostapp-extensions-update` but the script and service are called `update-hostapp-extensions`

Reply:
Yes. The recipe name is coherent with "hostapp-update", hence "hostapp-extension-update", and the script and service are coherent with the supervisor update as the mechanism is very similar, "update-resin-supervisor" and "update-hostapp-extension".
Because one is internal and one is external I don't see a problem in this, but I can change it if you feel strongly about it.